src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -101,28 +108,18 @@ def _webhook_event_data(
     return event_context
 
 
-@instrumented_task(name="sentry.sentry_apps.tasks.sentry_apps.send_alert_event", **TASK_OPTIONS)
+@instrumented_task(name="sentry.sentry_apps.tasks.sentry_apps.send_alert_event_v2", **TASK_OPTIONS)
```

Comment:
```suggestion
@instrumented_task(name="sentry.sentry_apps.tasks.sentry_apps.send_alert_webhook", **TASK_OPTIONS)
```

We could pick a name that doesn't have a version number here.